packages/client/hmi-client/src/workflow/ops/funman/tera-funman.vue

```diff
@@ -378,29 +387,16 @@ watch(
 </script>
 
 <style scoped>
```

Comment:
There is now potential spillage which looks a tad ugly

<img width="761" alt="image" src="https://github.com/DARPA-ASKEM/terarium/assets/17088680/0d169607-8106-41a8-85b5-6c990045b196">

Reply:
😢

Reply:
@Tom-Szendrey I think this should be good now, it's an issue with smaller screen size. I made that box more reactive to accommodate and a slightly smaller gap between inputs:

https://github.com/DARPA-ASKEM/terarium/assets/33158416/e3326994-f7b6-492f-97a5-44559195f066